Cookbook/Style-guide/README.md

```diff
@@ -460,7 +468,21 @@ func updateConstraints() -> () {
 typealias CompletionHandler = (result) -> ()
 ```
 
-## Function Calls
+For single-expression functions where the context is clear, both explicit and implicit returns can be used:
```

Comment:
This is similar to closures, do we have a preference to prefer no-return over return in closures? maybe then we should have the same preference here?

Reply:
AFAIK we have a preference for not using `return` in single expression closures.